daemon/logger/splunk/splunk.go

```diff
@@ -0,0 +1,256 @@
+// +build linux
```

Comment:
Yeah, if it works on Windows and you've verified it does, then remove line. But you'll need a line added to daemon\logdrivers_windows.go too to add splunk.